lib/scraper.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'httparty'
+require 'nokogiri'
+require_relative 'database'
+require_relative 'vacancy_database_handler'
+
+class Scraper
+  BASE_URL = 'https://openai.com'
+  START_URL = 'https://openai.com/careers/search'
+
+  def call
+    DataBase.connect
```

Comment:
extract all of this in initialise file

    DataBase.connect

    CreateVacancies.new.change unless ActiveRecord::Base.connection.table_exists?('vacancies')